core/templates/dev/head/pages/exploration_editor/editor_tab/StateEditor.js

```diff
@@ -30,6 +30,7 @@ oppia.controller('StateEditor', [
     $scope.currentStateIsTerminal = false;
     $scope.interactionIdIsSet = false;
     $scope.interactionIsShown = false;
+    $scope.lastAnswer = null;
```

Comment:
This is set on the StateEditor scope. I'm not sure that's the right thing to do, it's too far removed from the interaction preview.

Instead, when constructing the HTML tag for the interaction preview, could you construct it with lastAnswer = null or similar?

Reply:
Done, I am now setting lastAnswer attribute as null in ExplorationHtmlFormatterService, if in editor tab and all other cases, it gets actual lastAnswer value.